lms/djangoapps/badges/api/tests.py

```diff
@@ -0,0 +1,280 @@
+"""
+Tests for the badges API views.
+"""
+from django.conf import settings
+from django.test.utils import override_settings
+
+from badges.tests.factories import BadgeAssertionFactory, BadgeClassFactory, RandomBadgeClassFactory
+from util.testing import UrlResetMixin
+from xmodule.modulestore.tests.factories import CourseFactory
+from mobile_api.testutils import MobileAPITestCase, MobileAuthUserTestMixin
```

Comment:
Can we remove the interdependency between the badges app and the mobile_api one?  Since this app is generic enough and shouldn't be tied to mobile.
If there are items in the mobile_api project that are generally useful, feel free to refactor it and move it to a more common place that both apps can then use.